com.unity.render-pipelines.universal/Documentation~/universalrp-builtin-feature-comparison.md

```diff
@@ -115,56 +115,56 @@ This table provides an overview of the current features supported in the Univers
 | OnWillRenderObject                                           | Yes                                                          | Yes                                                          |
 | OnBecameVisible                                              | Yes                                                          | Yes                                                          |
 | OnBecameInvisible                                            | Yes                                                          | Yes                                                          |
-| Camera Replacement Material | Not supported     | Yes, with a custom Pass                               |
+| Camera Replacement Material                                  | Not supported                                                | Yes, with a custom Pass                                      |
 | RenderPipeline.BeginFrameRendering                           | Not supported                                                | Yes                                                          |
 | RenderPipeline.EndFrameRendering                             | Not supported                                                | Yes                                                          |
 | RenderPipeline.BeginCameraRendering                          | Not supported                                                | Yes                                                          |
 | RenderPipeline.EndCameraRendering                            | Not supported                                                | Yes                                                          |
 | UniversalRenderPipeline.RenderSingleCamera                   | Not supported                                                | Yes                                                          |
```

Comment:
This could be removed or said that also replaces Camera.Render, even though Camera.Render works in URP.